webview_panels/src/uiCore/components/button/Button.tsx

```diff
@@ -1,8 +1,31 @@
+import { useState } from "react";
 import { Button as ReactStrapButton, ButtonProps } from "reactstrap";
 import Tooltip from "../tooltip/Tooltip";
 
-export const Button = (props: ButtonProps): JSX.Element => (
-  <Tooltip title={props.title}>
-    <ReactStrapButton {...props} />
-  </Tooltip>
-);
\ No newline at end of file
+export const Button = (props: ButtonProps): JSX.Element => {
+  const defaultShowText = props?.showtext ? true : false;
```

Comment:
make sure the new props are defined explicitly. Other developers will not be able to find the props otherwise. They have to check the code to understand what is needed